docs/vehicle/state/vehicledatacombo.md

```diff
@@ -0,0 +1,285 @@
+      "charge_miles_added_rated": 127.5,
+      "charge_port_cold_weather_mode": null,
+      "charge_port_color": "Off",
+      "charge_port_door_open": false,
+      "charge_port_latch": "Blocking",
+      "charge_rate": 0.0,
+      "charger_actual_current": 0,
+      "charger_phases": null,
+      "charger_pilot_current": 24,
+      "charger_power": 0,
+      "charger_voltage": 0,
+      "charging_state": "Disconnected",
+      "conn_charge_cable": "<invalid>",
+      "est_battery_range": 69.59,
+      "fast_charger_brand": "<invalid>",
+      "fast_charger_present": false,
+      "fast_charger_type": "<invalid>",
+      "ideal_battery_range": 113.38,
+      "max_range_charge_counter": 1,
+      "minutes_to_full_charge": 0,
+      "not_enough_power_to_heat": false,
+      "off_peak_charging_enabled": false,
+      "off_peak_charging_times": "all_week",
+      "off_peak_hours_end_time": 360,
+      "preconditioning_enabled": false,
+      "preconditioning_times": "weekdays",
+      "scheduled_charging_mode": "Off",
+      "scheduled_charging_pending": false,
+      "scheduled_charging_start_time": null,
+      "scheduled_charging_start_time_app": 0,
+      "scheduled_departure_time": 1671166800,
+      "scheduled_departure_time_minutes": 360,
+      "supercharger_session_trip_planner": false,
+      "time_to_full_charge": 0.0,
+      "timestamp": 1685478699520,
+      "trip_charging": false,
+      "usable_battery_level": 44,
+      "user_charge_enable_request": null
+    },
+    "climate_state": {
+      "allow_cabin_overheat_protection": false,
+      "battery_heater": false,
+      "battery_heater_no_power": false,
+      "bioweapon_mode": false,
+      "cabin_overheat_protection": "Off",
+      "climate_keeper_mode": "off",
+      "cop_activation_temperature": "High",
+      "defrost_mode": 0,
+      "driver_temp_setting": 15.0,
+      "fan_status": 0,
+      "hvac_auto_request": "Override",
+      "inside_temp": 21.7,
+      "is_auto_conditioning_on": false,
+      "is_climate_on": false,
+      "is_front_defroster_on": false,
+      "is_preconditioning": false,
+      "is_rear_defroster_on": false,
+      "left_temp_direction": -160,
+      "max_avail_temp": 28.0,
+      "min_avail_temp": 15.0,
+      "outside_temp": 18.0,
+      "passenger_temp_setting": 15.0,
+      "remote_heater_control_enabled": false,
+      "right_temp_direction": -160,
+      "seat_heater_left": 0,
+      "seat_heater_rear_left": 0,
+      "seat_heater_rear_right": 0,
+      "seat_heater_right": 0,
+      "seat_heater_third_row_left": 0,
+      "seat_heater_third_row_right": 0,
+      "side_mirror_heaters": false,
+      "steering_wheel_heater": false,
+      "supports_fan_only_cabin_overheat_protection": false,
+      "timestamp": 1685478699520,
+      "wiper_blade_heater": false
+    },
+    "drive_state": {
+      "active_route_latitude": 0.0,
+      "active_route_longitude": 0.0,
+      "active_route_traffic_minutes_delay": 0.0,
```

Comment:
This can actually return more data if a destination is set in the navigation

```
            "active_route_destination": "Home",
            "active_route_energy_at_arrival": 73,
            "active_route_latitude": -1.123456,
            "active_route_longitude": 1.123456,
            "active_route_miles_to_arrival": 6.485299,
            "active_route_minutes_to_arrival": 23.466667,
            "active_route_traffic_minutes_delay": 0.0,
```